pkg/state/create.go

```diff
@@ -109,7 +109,7 @@ func (c *StateCreator) Parse(content []byte, baseDir, file string) (*HelmState,
 		if err == io.EOF {
 			break
 		} else if err != nil {
-			return nil, &StateLoadError{fmt.Sprintf("failed to read %s: reading document at index %d", file, i), err}
+			return nil, &StateLoadError{fmt.Sprintf("failed to read %s: reading document at index %d, you should add .gotmpl extension to your helmfile to enable helmfile to load it as a template since helmfile v1", file, i), err}
```

Comment:
Can we do this only when the file path misses the ".gotmpl" extension?

Also, a bit rewording would make this even nicer, like:

```suggestion
			return nil, &StateLoadError{fmt.Sprintf("failed to read %s: reading document at index %d. Started seeing this since Helmfile v1? Add the .gotmpl file extension", file, i), err}
```

Reply:
@mumoshu oh. good idea.